src/webgpu/api/operation/rendering/stencil.spec.ts

```diff
@@ -316,9 +310,15 @@ g.test('stencil_compare_func')
 g.test('stencil_passOp_operation')
   .desc(
     `
-    Test that each stencil operation works with the given stencil values correctly as expected.
-
-    TODO: Need to test failOp and depthFailOp as well.
+  Test that the stencil operation is executed on stencil pass. Triangle with stencil reference 3
+  passes the 'always' comparison function because the base stencil reference is 1. Then, test that
```

Comment:
The reader won't know what "base stencil reference" means until they know the details of how the test works. In this case since the comparison function is `'always'` I don't think it matters.
```suggestion
  Test that the stencil operation is executed on stencil pass.
  A triangle is drawn with the 'always' comparison function, so it passes. Then, test that
```